packages/tools/generator-widget/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - We've updated the version of yeoman-generator to 5.4.2.
 - We've updated the configurations for tsconfig in web widgets.
 - We've updated the npm tasks and template classes for web widgets.
+- You can now choose between generating class or function components.
```

Comment:
As the changes above were already published, you should move this line to above and tag the `## [9.1.0] - 2022-01-04` at the top of the previous changes. Sorry for that.

Reply:
like this?